test/makeup/lexers/diff_lexer_test.exs

```diff
@@ -29,13 +29,13 @@ defmodule Makeup.Lexers.DiffLexerTest do
       end
     end
 
-    property "lexting a string with an insertion" do
+    property "lexing a string with an insertion" do
       check all text <- inserted() do
         assert [{:generic_inserted, %{}, ^text} | _] = lex(text)
       end
     end
 
-    property "lexting a string with a deletion" do
+    property "lexing a string with a deletion" do
```

Comment:
That typo seems glaringly obvious in retrospect...

Reply:
Don't lext and drive! 🚗